_tools/linkcheck

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+check_link_file()
+{
+	if ! [ -f "${EXTERNAL_LINK_FILE}" ]; then
+		echo "The specified file $EXTERNAL_LINK_FILE does not exist. Exiting..."
+		exit 1
+	fi
+}
+
+
+check_urls()
+{
+	echo "Checking validity of external links..."
+
+	local URLS+=($(grep 'url:' $EXTERNAL_LINK_FILE | tail -n+1 | awk '{ print $2}'))
```

Comment:
you might want to add " around the file name, just to be sure paths with spaces will always be handled correctly